doc/source/release-notes/Percona-Server-8.0.25-15.1.rst

```diff
@@ -0,0 +1,54 @@
+.. _8.0.25-15.1:
+
+================================================================================
+*Percona Server for MySQL* 8.0.25-15.1
+================================================================================
+
+:Date: June 23, 2021
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/8.0/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 8.0.25-15.1
+includes all the features and bug fixes available in the
+`MySQL 8.0.24 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-24.html>`__ and
+the
+`MySQL 8.0.25 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-25.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+.. note::
+
+
+    The TokuDB Storage Engine will be disabled in future Percona Server for MySQL 8.0 versions. We recommend migrating to the MyRocks Storage Engine.
```

Comment:
Could you please also include a reminder that we already marked TokuDB as deprecated a while ago?
I provided a suggestion for this in https://jira.percona.com/browse/PS-7426

Reply:
Added.